src/components/Login/Login.js

```diff
@@ -14,9 +14,23 @@ class Login extends Component {
     this.setState({[e.target.name]: e.target.value})
   }
 
-  fetchUserData = (e) => {
+  checkUserData = (e) => {
     e.preventDefault();
-    fetch('https://rancid-tomatillos.herokuapp.com/api/v1/login',
+    this.fetchUserData()
+      .then(response => {
+        if(response.ok === true) {
+          this.fetchUserData()
+            .then(response => response.json())
+            .then(data => console.log(data))
+        } else {
+          alert("bad credentials");
```

Comment:
I'm totally fine with this being an alert, but if we have time we might want to make this a nicely-styled error message instead. Just another option!

Reply:
We will change it in future PR. Alert is a temporary indication if login credentials are bad.